### PR TITLE
All: add feed creation datetime

### PR DESCRIPTION
**Pull request by patochectp**

### CODE REVIEW

File: source/ed/connectors/fusio_parser.cpp [hunk 1]
```diff
@@ -106,7 +107,19 @@ void FeedInfoFusioHandler::init(Data&) {
...
+            feed_creation_time = boost::make_optional(boost::posix_time::duration_from_string(row[feed_info_value_c]));
+        } else {
+            data.add_feed_info(row[feed_info_param_c], row[feed_info_value_c]);
+        }
+    }
+}
+void FeedInfoFusioHandler::finish(Data& data) {
+    if (feed_creation_date && feed_creation_time) {
+        boost::posix_time::ptime creation_datetime(*feed_creation_date, *feed_creation_time);
+        data.add_feed_info("feed_creation_datetime",boost::posix_time::to_iso_string(creation_datetime));
```
Comment (kinnou02) [inline source/ed/connectors/fusio_parser.cpp:131]: spacing

File: source/ed/ed_reader.cpp [hunk 2]
```diff
@@ -282,6 +283,13 @@ void EdReader::fill_feed_infos(navitia::type::Data& data, pqxx::work& work){
         if (const_it["key"].as<std::string>() == "feed_license"){
             data.meta->license = const_it["value"].as<std::string>();
         }
+        if (const_it["key"].as<std::string>() == "feed_creation_datetime"){
+            try {
+                data.meta->dataset_created_at = bt::from_iso_string(const_it["value"].as<std::string>());
+            } catch(const boost::gregorian::bad_day_of_month&) {
```
Comment (kinnou02) [inline source/ed/ed_reader.cpp]: only the month can be bad?

File: source/ed/connectors/fusio_parser.cpp [hunk 3]
```diff
@@ -106,7 +107,19 @@ void FeedInfoFusioHandler::init(Data&) {
 
 void FeedInfoFusioHandler::handle_line(Data& data, const csv_row& row, bool) {
     if(is_valid(feed_info_param_c, row) && has_col(feed_info_value_c, row)) {
-        data.add_feed_info(row[feed_info_param_c], row[feed_info_value_c]);
+        if (row[feed_info_param_c] == "feed_creation_date"){
+            feed_creation_date = boost::make_optional(boost::gregorian::from_undelimited_string(row[feed_info_value_c]));
```
Comment (antoine-de) [inline source/ed/connectors/fusio_parser.cpp]: I think you should try/catch around this not to stop if the format is not right

File: source/navitia-proto [hunk 4]
```diff
@@ -1 +1 @@
-Subproject commit 6247e2c1b48f742016fd8946eee46d117ff031a6
+Subproject commit 40c5ccd07c3ee6a73928183bdcff296ed62ef8f7
```
Comment (kinnou02) [inline source/navitia-proto]: good: it's navitia-proto master head

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 5]
```diff
@@ -767,6 +767,7 @@ def output(self, key, obj):
     "is_open_data": fields.Boolean(),
     "is_realtime_loaded": fields.Boolean(),
     "realtime_proxies": fields.Raw(),
+    "dataset_created_at": fields.String(),
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py:770]: this should also be in /coverage no? poke @stifoon

**Comment by patochectp** [comment]
done

**Comment by kinnou02** [comment]
Jenkins, test this please.

**Pull request merged**